app/javascript/packs/userProfileSettings.js

```diff
@@ -1,12 +1,11 @@
-const userFieldIds = ['user[name]', 'user[email]', 'user[username]'];
+const userSettingForm = document.getElementById('user-profile-form');
 const profileFieldIds = Array.from(
   document.querySelectorAll('[id^="profile["]'),
 ).map((node) => node.id);
-const allFieldIds = [...userFieldIds, ...profileFieldIds];
 
 export function fieldCharacterLimits() {
-  window.addEventListener('load', () => {
-    allFieldIds.forEach((field_id) => {
+  if (document.readyState === 'complete') {
```

Comment:
what if you just removed this line (and the closing brace on line 25)? 

It seems like it behaves correctly - this script is loaded when we render the profile settings template, and the behavior for counting characters is an added listener.

Since the values for the input boxes aren't async (they're rendered in the template via the form builder) this seems like it works for both cases (Edit profile and "Settings" dropdown menu item).

Reply:
Just did this. Behaved alright in development. I appreciate a retest. 🙏🏾